src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js

```diff
@@ -147,7 +147,9 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
                 data: data.fileReader.result.replace(/^.*;base64,/, ''),
             };
 
-            if (data.file.type.startsWith('image/')) {
+            const contentType = response.ContentTypeInfoList.ContentType[0];
+
+            if (contentType.imageFields.identifier.includes(mapping.contentFieldIdentifier)) {
```

Comment:
Can we assign this condition to a variable for better readability? :) 
```suggestion
            const isImageContentType = contentType.imageFields.identifier.includes(mapping.contentFieldIdentifier)

            if (isImageContentType) {
```

Reply:
:+1:, but it seems that `isImageContentType` is not best name, as `ImageContentType` is usually related to `Image` content type, and from what i understand it is more `contentTypeContainsImageFieldType`